test/Ocelot.AcceptanceTests/Steps.cs

```diff
@@ -275,6 +289,62 @@ protected void StartOcelot(Action<WebHostBuilderContext, IConfigurationBuilder>
         _ocelotClient = _ocelotServer.CreateClient();
     }
 
+    /// <summary>
+    /// This is annoying cos it should be in the constructor but we need to set up the file before calling startup so its a step.
+    /// </summary>
+    /// <typeparam name="T">The <see cref="ILoadBalancer"/> type.</typeparam>
+    /// <param name="loadBalancerFactoryFunc">The delegate object to load balancer factory.</param>
+    public void GivenOcelotIsRunningWithCustomLoadBalancer<T>(Func<IServiceProvider, DownstreamRoute, IServiceDiscoveryProvider, T> loadBalancerFactoryFunc)
```

Comment:
Why was this method reintroduced? It should have no references as it is no longer utilized.
### Remove it❗

Reply:
Done!

Reply:
If it is done, feel free to press "Resolve" button. If you want to highlight resolving commit then write a hash of the commit.